acoustid/future/fpindex/client.py

```
@@ -79,6 +79,40 @@ class FingerprintIndexClientError(Exception):
 
+class BatchUpdate:
+    def __init__(self):
+        self.changes: list[Change] = []
+
+    def insert(self, id: int, hashes: list[int]) -> None:
+        self.changes.append(
+            Change(
+                insert=Insert(
+                    id=id,
+                    hashes=hashes,
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)